src/main/java/org/jboss/pull/shared/connectors/bugzilla/Bug.java

```diff
@@ -46,10 +47,10 @@
     private static final long serialVersionUID = 6967220126171894474L;
 
     // includes attributes for Bug.get execution
-    public static final Object[] include_fields = { "id", "alias", "last_change_time" , "product", "component", "version", "priority",
-            "severity", "target_milestone", "creator", "assigned_to", "qa_contact", "docs_contact", "status", "resolution",
-            "flags", "groups", "depends_on", "blocks", "target_release", "summary", "description", "cf_type", "creation_time",
-            "estimated_time", "actual_time", "remaining_time" };
+    public static final Object[] include_fields = { "id", "alias", "last_change_time", "product", "component", "version",
+            "priority", "severity", "target_milestone", "creator", "assigned_to", "qa_contact", "docs_contact", "status",
+            "resolution", "flags", "groups", "depends_on", "blocks", "target_release", "summary", "description", "cf_type",
+            "creation_time", "estimated_time", "actual_time", "remaining_time", "external_bugs" };
```

Comment:
Does this still work when calling params.put("include_fields", Bug.include_fields); inside BugsClient.getBug()?  I did not check the redhat bugzilla api document, but it should be fine if "external_bugs" and its attributes are part of 'get' action returns.